response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -63,3 +64,37 @@ def update_collection_exercise_details(collection_exercise_id, user_description,
         raise ApiError(response)
 
     logger.debug('Successfully updated collection exercise details', collection_exercise_id=collection_exercise_id)
+
+
+def create_collection_exercise(survey_id, survey_name, user_description, period):
+    logger.debug('Creating a new collection exercise for', survey_id=survey_id, survey_name=survey_name)
+    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/collection-exercise/create-collection-exercise'
```

Comment:
Could just go directly to collection exercise service now, the back stage endpoint provide no additional functionality here.

Reply:
Yeah I finished all this before people started talking about not using backstage 😢

Reply:
Probably best to close the backstage PR now.